Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -328,32 +334,30 @@ theorem choose_exists_companion : Q.exists_companion.choose = polarBilin Q :=
       add_sub_cancel_left]
 
 protected theorem map_sum {ι} [DecidableEq ι] (Q : QuadraticMap R M N) (s : Finset ι) (f : ι → M) :
-    Q (∑ i ∈ s, f i) = ∑ i ∈ s, Q (f i) +
-      ∑ ij ∈ s.sym2 with ¬ ij.IsDiag,
-        Sym2.lift ⟨fun i j => polar Q (f i) (f j), fun _ _ => polar_comm _ _ _⟩ ij := by
+    Q (∑ i ∈ s, f i) = ∑ i ∈ s, Q (f i)
+      + ∑ ij ∈ s.sym2 with ¬ ij.IsDiag, ((polar_sym2 Q) ∘ Sym2.map f) ij := by
```

Comment:
Any reason you're not writing this as
```suggestion
      + ∑ ij ∈ s.sym2 with ¬ ij.IsDiag, polar_sym2 Q (ij.map f) := by
```
?

Reply:
I tried `((polar_sym2 Q) ∘ (ij.map f))` which was wrong.

Reply:
I guess it matched `linearCombination R ((polarSym2 Q) ∘ Sym2.map g)` in #18578.